os-brainux/setup_brainux.sh

```diff
@@ -70,14 +70,14 @@ systemctl enable fake-hwclock
 # Ly
 apt install -y libpam0g-dev libxcb-xkb-dev
 cd /
-git clone https://github.com/nullgemm/ly.git
+git clone --recurse -submodules https://github.com/nullgemm/ly.git
```

Comment:
(Nits) ハイフン1つの表記は解釈に曖昧さがあるため、ハイフン2つに統一してください

```suggestion
git clone --recurse --submodules https://github.com/nullgemm/ly.git
```